src/main/java/uk/co/evoco/metrics/MetricRegistryHelper.java

```diff
@@ -0,0 +1,51 @@
+package uk.co.evoco.metrics;
+
+import com.codahale.metrics.MetricFilter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.graphite.Graphite;
+import com.codahale.metrics.graphite.GraphiteReporter;
+import com.codahale.metrics.jmx.JmxReporter;
+import uk.co.evoco.webdriver.configuration.TestConfigHelper;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.TimeUnit;
+
+public class MetricRegistryHelper {
+
+    private static MetricRegistryHelper metricRegistryHelper;
+    private static MetricRegistry metricRegistry;
+    private static JmxReporter jmxReporter;
+    private static GraphiteReporter graphiteReporter;
+    private static final Graphite graphite = new Graphite(new InetSocketAddress("localhost", 2003));
+
+    /**
+     *
+     */
+    private MetricRegistryHelper() {
+        this.metricRegistry = new MetricRegistry();
+        if(TestConfigHelper.get().getMetricsConfig().getJmxConfig().isEnabled()) {
+            jmxReporter = JmxReporter.forRegistry(metricRegistry).build();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/gh/digital-delivery-academy/selenium-pom-framework/pullRequest?prid=5230970)